testPI2/unit/shared/setup.test.js

```diff
@@ -95,6 +95,7 @@ Test('setup', setupTest => {
         test.ok(Sidecar.connect.calledWith(service))
         test.notOk(Migrator.migrate.called)
         test.equal(s, server)
+        Sidecar.connect.restore()
```

Comment:
why did you change this from Sidecar.connect = sandbox.stub()?

Reply:
This was causing the tests to fail in admin/participants/handler.test.js .. since the Sidecar.connect was not being restored